doc/source/user_guide/groupby.rst

```diff
@@ -1264,6 +1264,7 @@ Categorical variables represented as instance of pandas's ``Categorical`` class
 can be used as group keys. If so, the order of the levels will be preserved:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
these are the ones that need to go. the problem is that this shows for *any* groupby that happens to hit this path, so i think users are going to have to be explicit on the keyword

Reply:
Yeah, I just missed this one.

Reply:
Fixed